build.gradle

```diff
@@ -585,6 +585,10 @@ dependencies {
     implementation 'com.nimbusds:nimbus-jose-jwt:9.41.2'
     implementation 'com.rfksystems:blake2b:2.0.0'
     implementation 'com.password4j:password4j:1.8.2'
+
+    // Action privileges: check tables and compact collections
+    implementation 'com.selectivem.collections:special-collections-complete:1.4.0'
```

Comment:
Is this the source for this lib? https://github.com/nibix/collections

Nice to see 95% coverage!

Reply:
Yep 👍😃